CODEC/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM --platform=linux/amd64 ubuntu:20.04
+
+LABEL maintainer="lining@broadinstitute.org"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update \
+    && apt-get install -y software-properties-common \
+    && add-apt-repository ppa:deadsnakes/ppa \
+    && apt-get update \
+    && apt-get install -y python3.8 python3.8-dev python3.8-venv python3-pip \
+    && pip3 install pandas argparse numpy pysam
+
+RUN apt-get install -y r-base r-base-dev
+
+RUN apt-get install -y \
+    git \
+    wget \
+    bwa \
+    libssl-dev \
+    g++ \
+    zlib1g-dev \
+    autoconf \
+    libbz2-dev \
+    liblzma-dev \
+    libcurl4-gnutls-dev \
+    libssl-dev \
+    build-essential \
+    software-properties-common 
+
+RUN apt-get update -qq
+RUN apt-get install -y openjdk-11-jdk
+
+
+
+# Clone the CODECsuite repository
+RUN git clone --recursive https://github.com/broadinstitute/CODECsuite.git /CODECsuite
```

Comment:
Separate CODEC Docker from the rest of Docker images that were used in the CODEC WDL.